### DISCUSSION TITLE
Stirling-PDF in Home Assistant

### DISCUSSION THREAD

**Discussion by ndsrf** [alexbelgium/hassio-addons#1035] [Ideas]
It will be fantastic to self-host a PDF swiss knife like this one: https://github.com/Frooodle/Stirling-PDF

**Comment by TechSaiyajin** [comment]
A very good idea!

https://github.com/Frooodle/Stirling-PDF

**Comment by scheusal333** [comment]
i want to give this a push aswell

**Comment by pete-gh** [comment]
Yes please.

**Comment by pablodelarco** [comment]
Hi guys, I developed a custom Stirling-PDF add-on for Home Assistant: https://github.com/pablodelarco/stirling_pdf_addon

Hope you like it ! :)

@pete-gh @scheusal333 @TechSaiyajin @ndsrf

**Reply by alexbelgium** [reply]
Thanks for sharing! Congrats with the addon
